cascade-api/src/Cascade/Api/Network/Anatomy/Api/Projects.hs

```diff
@@ -26,22 +19,16 @@ import           Cascade.Api.Network.Anatomy.Prelude
 import qualified Cascade.Api.Servant.Response       as Response
 import           Data.Generics.Labels                ( )
 
-type CreateResponse = '[Response.Created Project.Readable]
+type GetByIdResponse = '[Response.Unauthorized , Response.Forbidden , Response.NotFound , Response.Ok Project.Readable]
 
-type GetAllResponse = '[Response.Ok [Project.Readable]]
+type UpdateByIdResponse = '[Response.Unauthorized , Response.Forbidden , Response.NotFound , Response.Ok Project.Readable]
 
-type GetByIdResponse = '[Response.Ok Project.Readable , Response.NotFound]
-
-type UpdateByIdResponse = '[Response.Ok Project.Readable , Response.NotFound]
-
-type DeleteByIdResponse = '[Response.Ok Project.Readable , Response.NotFound]
+type DeleteByIdResponse = '[Response.Unauthorized , Response.Forbidden , Response.NotFound , Response.Ok Project.Readable]
```

Comment:
I think Response.Forbidden in these three responses is redundant, they never respond with that.